hosting/src/components/Header/DropdownUser/components/SignOutWithGoogle.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+import {removeSession} from "@/actions/auth-action";
+import {signOutWithGoogle} from "@/libs/firebase/auth";
+
+export const SignOutWithGoogle = () => {
```

Comment:
Could we rename the operation to just "sign out" since the sign out flow will be the same for all providers when we add more in the future.